canonical-web-and-design/vanilla-framework
# Update ./run script to latest version

## Done

Updates ./run script

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/
- Make sure everything runs as before
